lib/bfd.c

```diff
@@ -840,10 +841,14 @@ void bfd_sess_show(struct vty *vty, struct json_object *json,
 	}
 
 	bfd_last_update(bsp->bss.last_event, time_buf, sizeof(time_buf));
+	epoch_tbuf = time(NULL) - (monotime(NULL) - bsp->bss.last_event);
 	if (json) {
 		json_object_string_add(json_bfd, "status",
 				       bfd_get_status_str(bsp->bss.state));
 		json_object_string_add(json_bfd, "lastUpdate", time_buf);
+		json_object_int_add(json_bfd, "bfdLastUpdateEpoch", epoch_tbuf);
+		json_object_string_add(json_bfd, "bfdLastUpdateEpochStr",
+				       ctime(&epoch_tbuf));
```

Comment:
no `ctime` - must use `ctime_r`

Reply:
Should we add that to frrbot's list of banned functions?

Reply:
wow - a great idea! yes, I do think so - is that list in our source somewhere, or is it out in github-land somewhere?

Reply:
https://github.com/FRRouting/frrbot/blob/master/frrbot.py#L29

Reply:
super - done: https://github.com/FRRouting/frrbot/pull/8